tests/_support/Helper/UserManagement.php

```diff
@@ -110,7 +110,7 @@ protected function ensureUserExistsWithRoleLocal(string $role, $status): UserMan
             ->getQuery()
             ->getOneOrNullResult();
 
-        if ($user) {
+        if ($user && !is_null($status)) {
```

Comment:
It might be better to try finding a user with the correct status in the query getting the user than to always create a new user when the status is null (should `null` even be allowed? or should it default to a specific status?).